docs/installation.rst

```diff
@@ -99,6 +99,12 @@ Add django-debug-toolbar's URLs to your project's URLconf:
 This example uses the ``__debug__`` prefix, but you can use any prefix that
 doesn't clash with your application's URLs.
 
+> [!NOTE]
+> Check  out the configuration example in the
+[example app](https://github.com/jazzband/django-debug-toolbar/tree/main/example)
+to learn how to set up the toolbar to function smoothly while running your
+tests.
```

Comment:
We're using restructured text which has a different syntax than markdown. You'll want to use a note admonition which you can read about here: https://docutils.sourceforge.io/docs/ref/rst/directives.html#note